src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -71,39 +77,46 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         switch (column.key) {
             case 'properties':
                 return (
-                    <IconButton
-                        iconProps={{ iconName: 'EntryView' }}
-                        title={t('advancedSearch.inspectProperty')}
-                        ariaLabel={t('advancedSearch.inspectProperty')}
-                        onClick={(event) => {
-                            event.stopPropagation();
-                        }}
-                        className={'cb-scenes-action-button'}
+                    <PropertyInspectorCalloutButton
+                        twinId={`${currentTwin?.$dtId}`}
+                        adapter={adapter}
                     />
                 );
             default:
                 return <span>{fieldContent}</span>;
         }
     };
+    const selection = new Selection({
+        getKey(item, index?) {
+            return item + index;
+        },
+        onSelectionChanged: () => {
+            setTwin(selection.getSelection());
+            onTwinSelection(currentTwin);
+        }
+    });
 
     return (
         <section className={classNames.root}>
-            <h3>{t('advancedSearch.results', { twinCount })}</h3>
+            <h3 className={classNames.headerCorrection}>
+                {t('advancedSearch.results', { twinCount })}
+            </h3>
             <DetailsList
                 items={twins}
                 columns={
                     additionalColumns.length > 0
                         ? columns.concat(additionalColumns)
                         : columns
                 }
-                layoutMode={DetailsListLayoutMode.fixedColumns}
+                layoutMode={DetailsListLayoutMode.justified}
```

Comment:
I havne't used this prop before. What made you change to justified?

Reply:
This occurs when it's fixed width

![image](https://user-images.githubusercontent.com/80120000/186010517-a9725ccb-cba6-4384-acdd-617dee22c1e4.png)